src/map-styler-hook/useMapStyler.js

```diff
@@ -1,27 +1,35 @@
 import { useEffect, useState } from 'react'
 
-function useMapStyler(layerComposer, layers) {
+const applyStyleTransformations = (style, styleTransformations) => {
+  let newStyle = style
+  styleTransformations.forEach((t) => {
+    newStyle = t(newStyle)
```

Comment:
Great! I also like this idea of compose hooks https://codesandbox.io/s/github/tannerlinsley/react-table/tree/master/examples/sorting from react-table so we could have something similar to the different transformations